src/app/play-round/play-round.component.html

```diff
@@ -1,10 +1,10 @@
-<app-top-menu-bar [contents]="contents"></app-top-menu-bar>
```

Comment:
There seem to be a couple of places where you are removing attributes similar to this one. Is that going to make testing more difficult?

Reply:
Oh, no, that attribute was just a quirk of the old implementation :stuck_out_tongue: We were using it to make the toolbar float, but now Paul's doing that a different (better) way.